Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -34,8 +34,18 @@ fi
 # shellcheck disable=SC2034
 if [[ -n "${ruby_SHA}" && -n "${ruby_FILENAME}" ]]
 then
-  ruby_URL="https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:${ruby_SHA}"
-  ruby_URL2="https://github.com/Homebrew/homebrew-portable-ruby/releases/download/2.6.3_2/${ruby_FILENAME}"
+  ruby_URLs=()
+  if [[ -n "${HOMEBREW_ARTIFACT_DOMAIN}" ]]
+  then
+    ruby_URLs+=("${HOMEBREW_ARTIFACT_DOMAIN}/bottles-portable-ruby/${ruby_FILENAME}")
```

Comment:
The documentation for `HOMEBREW_ARTIFACT_DOMAIN` is outdated. It's a now mirror for ghcr.io URLs only, so I think this one should be: `${HOMEBREW_ARTIFACT_DOMAIN}/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:${ruby_SHA}`.

`HOMEBREW_BOTTLE_DOMAIN` is correct.